openquakeplatform/pla_common/management/commands/add_documents.py

```diff
@@ -367,13 +378,33 @@ def handle(doc_fname, *args, **options):
             title_en='isc_viewer_measure').exclude(
             title_en='ghec_viewer_measure').delete()
 
+        # Import SpatialRepresentationType
+        srt_old_refs = {}
+        for srt_full in srt_load:
+
+            field = srt_full['fields']
+
+            new_srt = SpatialRepresentationType.objects.model(
+                is_choice=field['is_choice'],
+                gn_description=field['gn_description'],
+                identifier=field['identifier'],
+                description=field['description']
+                )
```

Comment:
if field fields names are the same and are only that names you can use
``new_srt = SpatialRepresentationType.objects.model(**field)``